lib/rb_inject.rb

```diff
@@ -0,0 +1,48 @@
+class Array
+
+  def inject_custom(accumulator = nil, operation = nil, &block)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for inject_custom is too high. [21.21/15]